### DISCUSSION TITLE
New (Automated) BirdWeather ID Request Form

### DISCUSSION THREAD

**Discussion by timsterc** [Nachtzuster/BirdNET-Pi#66] [General]
Hi Everyone!

With an increasing number of BirdWeather ID requests coming through - we've decided to make it an automated process that you're able to do yourself.

You’ll first need to create an account - https://app.birdweather.com/login

Once that’s done - you can go to - https://app.birdweather.com/account/stations
Make sure that the Latitude and Longitude match what is in your BirdNET-Pi configuration.

NOTE - by using your BirdWeather ID - you are consenting to sharing your soundscapes and detections with BirdWeather.

Cheers!
-Tim
Founder - BirdWeather

**Comment by alexbelgium** [comment]
That's incredible, thanks very much!!!
Can I link my existing ID to my new account to avoid creating a duplicate?

**Reply by timsterc** [reply]
@alexbelgium - I can make the link on the backend.  You'd just need to email me with your user email and Station token (support@birdweather.com)

**Reply by lloydbayley** [reply]
Very good of you Tim, sorry to create more work for you.

**Reply by timsterc** [reply]
I should add ... if I get hundreds of email requests to add existing stations (which really doesn't give you any additional functionality by the way) ... that you may need to be patient.

**Reply by lloydbayley** [reply]
Fair-play to you. I was just concerned that it might need to be something we already-established people might have to do to make sure things were up to date and made your life at BirdWeather easier.
If this does not NEED to be done by existing users, then it will be up to the individual!
Thanks again!

**Comment by lloydbayley** [comment]
Thanks for letting us know Tim, I've done a PR to make the required changes to the config page.
Also, @alexbelgium has a good point... Should we already-established users do something to create an account or should we just let sleeping dogs lie, as we already have our IDs?

**Comment by Hiemstra87** [comment]
@timsterc, I was wondering why there was no reply to my mail. I'll apply through the new way.

Thanks!

**Comment by Nachtzuster** [comment]
@timsterc Thanks!
the PR has been merged

**Comment by wenestvedt** [comment]
How long should it take before our sightings show up on Birdweather (assuming that our station appears on the map, and has been linked with the token)?

**Comment by philwhitt** [comment]
for my station if I searched for a species I saw hits at my location almost immediately. For the station to show up on the map without searching took a couple of hours

**Reply by wenestvedt** [reply]
Excellent, thank you!

**Comment by dalogue1** [comment]
My station showed up on the map. Virtually right away.

![image](https://github.com/Nachtzuster/BirdNET-Pi/assets/133064876/79be8e5e-6e06-4a61-9f93-e2f0a4dff396)

![image](https://github.com/Nachtzuster/BirdNET-Pi/assets/133064876/32ebf349-30b1-4a15-9232-9a932f5f4bd1)

**Reply by alexbelgium** [reply]
It's probably waiting for you to have more hits ; I think there is a threshold of number of detections before it starts showing them

**Reply by dalogue1** [reply]
It showed up when I hit refresh.

**Comment by dalogue1** [comment]
If you delete a detection does it remove it from BirdWeather.  If not, how can you remove a detection that was bad or faint from BirdWeather?

Otherwise how can you keep every bad detection from being posted?  Maybe an approval feature is needed.

**Reply by alexbelgium** [reply]
No there is no such options - faint detections are noted in a specific way in birdweather I think as some observations seem to have a green/orange/red color code next to them, but we can't delete or modify them. I think their strategy is quantity more than quality, in the sense that statistically the true species with emerge on top and erroneous observations will remain scarce. On our systems however, playing with detection level, sensibility, species threshold, species exclusions (and hopefully whitelist) will allow us to improve over time the quality of data we send to birdweather

**Comment by dalogue1** [comment]
Do you have a BirdWeather station number?  How do you manage bad detections if you do?

**Reply by alexbelgium** [reply]
Mine is here! https://app.birdweather.com/stations/4296 BTW it's not my exact address and location I'm a bit paranoiac with that (who knows if someone REALLY angry about my false detections would come at my home ;) ).

Indeed false detections bugged me ; and wrong detections. I've tried several settings but in the end settled in v2.4 model (the other 6k model is more precise in other parts of the world) ; with species threshold at 0.007 to detect night birds ; why sensitivity at 1.25 (default). Each rare bird triggers a message on my phone through apprise, and based on audio confirmation I add the specie to my exclude list or not. I was actually surprised about some species that I did not thought present but actually was able to confirm visually. I'm now more prudent about exclusions.

In the end, this means that indeed I have false positives uploaded to birdweather but it really is a very low amount and quality will increase over time...

**Comment by dalogue1** [comment]
Is this going to post the Eastern Gray Squirrel?  I like when my station detects them but I don't know if they want squirrel sightings posted on their website.

**Reply by alexbelgium** [reply]
Hi, the model detects non birds (frogs, engines...) but ignores them! Specific models exist for cricket or bats but then the app must be modified

**Reply by dalogue1** [reply]
It added the squirrel to my detections.  Next time I have a squirrel what is going to stop it from being reported?

![image](https://github.com/Nachtzuster/BirdNET-Pi/assets/133064876/e0866b46-a4f1-4b5e-95f9-5d004fca5aa2)

**Reply by alexbelgium** [reply]
??? Ok then I thought wrong. Indeed only solution is to add it to the exclude list...

**Comment by dalogue1** [comment]
posted new thread
